akka-docs/src/main/paradox/typed/event-stream.md

```diff
@@ -0,0 +1,289 @@
+# EventStream
+
+You are viewing the documentation for the new actor APIs, to view the Akka Classic documentation, see @ref:[Classic Event Stream](../event-bus.md#event-stream).
+
+## Dependency
+
+<!--- #dependency --->
+The Akka dependencies are available from Akka's library repository. To access them there, you need to configure the URL for this repository.
+
+@@repository [sbt,Maven,Gradle] {
+id="akka-repository"
+name="Akka library repository"
+url="https://repo.akka.io/maven"
+}
+
+To use Akka Actor Typed, you must add the following dependency in your project:
+
+@@dependency[sbt,Maven,Gradle] {
+bomGroup=com.typesafe.akka bomArtifact=akka-bom_$scala.binary.version$ bomVersionSymbols=AkkaVersion
+symbol1=AkkaVersion
+value1="$akka.version$"
+group=com.typesafe.akka
+artifact=akka-actor-typed_$scala.binary.version$
+version=AkkaVersion
+}
+<!--- #dependency --->
+
+## Introduction
+
+The event stream is the main @ref:[Event Bus](#eventbus). of each actor system: it is used for
+carrying @ref:[log messages](logging.md) and @ref:[Dead Letters](#dead-letters) and may be
+used by the user code for other purposes as well. 
+
+It uses @ref:[Subchannel Classification](#subchannel-classification) which enables registering to related sets of channels.
+
+
+## How to use
+
+The following example demonstrates how a simple subscription works. Given a simple actor:
+
+@@@ div { .group-scala }
+
+@@snip [LoggingDocSpec.scala](/akka-actor-typed-tests/src/test/scala/akka/actor/typed/eventstream/LoggingDocSpec.scala) { #deadletters }
+
+@@@
+
+@@@ div { .group-java }
+
+@@snip [LoggingDocTest.java](/akka-actor-typed-tests/src/test/java/akka/actor/typed/eventstream/LoggingDocTest.java) { #imports-deadletter }
+
+@@snip [LoggingDocTest.java](/akka-actor-typed-tests/src/test/java/akka/actor/typed/eventstream/LoggingDocTest.java) { #deadletter-actor }
+
+it can be subscribed like this:
+
+@@snip [LoggingDocTest.java](/akka-actor-typed-tests/src/test/java/akka/actor/typed/eventstream/LoggingDocTest.java) { #deadletters }
+
+@@@
```

Comment:
You are correct, however, this `div { group-java/scala }` block will still follow tab switching. 

Therefore, we will add a Tabbed to the first snippet of this block and retain the `div { group-java }` only, as it is not allowed to combine imports and class definitions in Java (it would be broken by the format plugin).

The final result is like this on my PC :), I think this will be better.

![截屏2023-10-10 10 10 54](https://github.com/akka/akka/assets/26020358/8bea8cde-a52a-48a8-a1ea-3ef28191e7c4)

Reply:
Ok, that is better, the big problem was the first code snippet in the page not having the tabs.